From f4e5a3f0c0cf055d40ec252aba929d6da13ffbfb Mon Sep 17 00:00:00 2001
From: Piyush Mehta <piyush.mehta@xilinx.com>
Date: Wed, 15 Jul 2020 12:30:51 +0530
Subject: [PATCH 1444/1852] usb: dwc3: Remove unused variable and values
 warnings

commit 8706f4eb4d1ee2b9073c926f72fe4b16d87bb64f from
https://github.com/Xilinx/linux-xlnx.git

This patch removes the unused variables and values.

Reported Coverity warning: UNUSED_VALUE

Signed-off-by: Piyush Mehta <piyush.mehta@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/usb/dwc3/gadget.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/usb/dwc3/gadget.c b/drivers/usb/dwc3/gadget.c
index ce63c15e13a6..fedaff3ff7ee 100644
--- a/drivers/usb/dwc3/gadget.c
+++ b/drivers/usb/dwc3/gadget.c
@@ -3651,12 +3651,10 @@ int dwc3_gadget_init(struct dwc3 *dwc)
 					dev_err(dwc->dev,
 						"otg_set_peripheral failed\n");
 					usb_put_phy(phy);
-					phy = NULL;
 					goto err4;
 				}
 			} else {
 				usb_put_phy(phy);
-				phy = NULL;
 			}
 		}
 	}
-- 
2.31.1

